.travis.yml

```diff
@@ -30,20 +24,20 @@ jobs:
     templates:
         - &test-matrix
           stage: Test
-          after_success: beaver dlang codecov
           script:
               - set -e # Do not continue on failure
-              - beaver dlang install
-              - beaver dlang make
-        - &package-matrix
-          stage: Build and upload packages
-          if: tag IS present
+              - |
+                docker build --build-arg DMD=$DMD --build-arg DIST=$DIST \
+                --build-arg F=$F \
+                -t tester -f docker/Dockerfile.tester .
+              - docker run --rm -it tester
+        - &container-matrix
+          stage: Build a docker container
           script:
               - set -e # Do not continue on failure
-              - beaver dlang install
-              - beaver dlang make
-              - beaver dlang make pkg
-              - beaver bintray upload -d sociomantic-tsunami/nodes/dhtnode build/production/pkg/*.deb
+              - |
+                docker build --build-arg DMD=$DMD --build-arg DIST=$DIST \
+                -t dhtnode -f docker/Dockerfile.tester .
```

Comment:
Shouldn't one of `container-matrix` or `test-matrix` use `docker/Docker.dhtnode`? If not, what is that file for?